corehq/pillows/app_submission_tracker.py

```diff
@@ -126,3 +131,89 @@ def get_sql_app_form_submission_tracker_reindexer():
         FormReindexAccessor(include_attachments=False)
     )
     return AppFormSubmissionReindexer(doc_provider, FORM_SQL, 'form_processor_xforminstancesql')
+
+
+@quickcache(['domain'], timeout=60 * 60 * 5)
+def _get_apps_for_domain(domain):
+    project = Domain.get_by_name(domain)
+    if project:
+        return [app._id for app in project.applications()]
```

Comment:
Also since you're only wanting the app IDs you can just use `get_app_ids_in_domain`